pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/v2/PNConfiguration.kt

```diff
@@ -479,6 +504,10 @@ interface PNConfiguration : com.pubnub.api.v2.PNConfiguration {
          *
          * @see [HttpLoggingInterceptor]
          */
+        @Deprecated(
+            message = "This setting is deprecated. Use customLoggers  instead",
```

Comment:
The same nitpick as above, there's an extra space between `customLoggers` and `instead`